ui/src/main/java/org/pentaho/di/ui/job/entries/pgpencryptfiles/JobEntryPGPEncryptFilesDialog.java

```diff
@@ -520,7 +557,7 @@ public void widgetSelected( SelectionEvent e ) {
     wSourceFileFolder.addModifyListener( lsMod );
     FormData fdSourceFileFolder = new FormData();
     fdSourceFileFolder.left = new FormAttachment( middle, 0 );
-    fdSourceFileFolder.top = new FormAttachment( wSettings, 2 * margin );
+    fdSourceFileFolder.top = new FormAttachment( wAction, margin );
```

Comment:
Margin transformation (2 * margin) was removed on line 560 but kept on 501.  Is this Intentional?

Reply:
good catch, the label was barely noticeable off center. It will be corrected with the next push.